Makefile

```diff
@@ -35,6 +35,10 @@ lint:
 	go vet ${PACKAGES}
 	gofmt -d -l ${GOFILES}
 
+# Install with go get honnef.co/go/tools/cmd/staticcheck
```

Comment:
Would probably prefer to rely on standard tooling for static analysis (c.f., L35 of this file) instead of an externally installed dep, with no automation in the Makefile to check for existence (or install it).

Will defer to @broamski

Reply:
I can either add automation to check for existence or remove entirely. Let me know.
I've made only good experiences with `staticcheck` in the past.